### PR TITLE
fix: validations in manifests ace/aci

### PR DESCRIPTION
**Pull request by Hardeep8587**
## Change Description
validations in ace/aci
extra comonents deleted 
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Hardeep8587** [comment]
@kunalfleethawks @amritfleethawks Please review and merge

File: src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.ts [hunk 1]
```diff
@@ -758,7 +758,7 @@ export class NewAciManifestComponent implements OnInit {
         this.response = res;
         this.hasSuccess = true;
         this.toastr.success('Manifest Updated Successfully');
-        this.router.navigateByUrl('/dispatch/cross-border/eManifests');
+        this.location.back(); // <-- go back to previous location
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.ts:759]: Manifest Updated Successfully (add a period after message)

**Review by kunalfleethawks — APPROVED** [review]

**Review by amritfleethawks — APPROVED** [review]

File: src/app/pages/dispatch/new-documents/company-documents/company-documents.component.ts [hunk 2]
```diff
@@ -47,13 +48,13 @@ export class CompanyDocumentsComponent implements AfterViewInit, OnDestroy, OnIn
   Success: string = '';
   errors = {};
   carrierID: any;
-  documentData = {
+  documentData = {=
```
Comment (amritfleethawks) [inline src/app/pages/dispatch/new-documents/company-documents/company-documents.component.ts]: hardeep please remove '='

**Review by amritfleethawks — CHANGES_REQUESTED** [review]

File: src/app/pages/dispatch/new-documents/company-documents/company-documents.component.ts (hunk 2 above)
Comment (Hardeep8587) [reply src/app/pages/dispatch/new-documents/company-documents/company-documents.component.ts]: done !

**Review by amritfleethawks — APPROVED** [review]

**Pull request merged**